Libbna/CUSTOM-CMS
# #275: Fixed-user-notfound-error

**Testing:**

- Open the Login form.
- Add a username that is not in DB.
- Add a password.
- A Error msg will appear "Username not found" after clicking on login button.
- Refer to the following screenshot.

![image](https://user-images.githubusercontent.com/81369580/154243294-28040712-1e0f-489b-bc9e-4547dabb2756.png)
